docs/source/projects/architecture/content-architecture.rst

```diff
@@ -64,10 +72,12 @@ based on the same kernel, but it also made it possible to support new
 languages in the same frontends, by developing kernels in those languages, and
 we are refining IPython to make that more practical.
 
-Today, there are two ways to develop a kernel for another language. Wrapper
-kernels reuse the communications machinery from IPython, and implement only
+Today, there are three ways to develop a kernel for another language. Wrapper
+kernels reuse the communications machinery from IPykernel, and implement only
 the core execution part. Native kernels implement execution and communications
-in the target language.
+in the target language. Kernels based on `xeus <https://github.com/jupyter-xeus/xeus>`_,
+a native implementation of the protocol, implement the language-specific part of the
+kernels. Contrary to the wrapper approach, `xeus` does not depend on a python runtime.
```

Comment:
I'd suggest turning this into a three-bullet list so that each is easier to notice. e.g.:

```
- **Wrapper kernels** are ...
- **Native kernels** implement ...
- **xeus-based kernels** implement ...
```